Code/Framework/AzManipulatorTestFramework/Include/AzManipulatorTestFramework/AzManipulatorTestFrameworkTestHelpers.h

```diff
@@ -52,8 +52,9 @@ namespace UnitTest
 
     //! Fixture to provide the indirect call viewport interaction that inherits from ToolsApplicationFixture for the
     //! dependent on AzToolsFramework::ToolsApplication.
+    template<bool CheckForLeaksOnDestruction = true>
```

Comment:
Does `IndirectCallManipulatorViewportInteractionFixtureMixin` fixture ever get used with a value of false?
If not, then the template parameter isn't needed.

Reply:
Fixed.